packages/react-devtools-shared/src/devtools/views/Profiler/SidebarSelectedFiberInfo.js

```diff
@@ -31,12 +31,68 @@ export default function SidebarSelectedFiberInfo(_: Props) {
     selectFiber,
   } = useContext(ProfilerContext);
   const {profilingCache} = profilerStore;
+  const listContainer = useRef<HTMLElement | null>(null);
+  const listItemRef = useRef<HTMLElement | null>(null);
 
   const commitIndices = profilingCache.getFiberCommits({
     fiberID: ((selectedFiberID: any): number),
     rootID: ((rootID: any): number),
   });
 
+  const handleKeyDown = useCallback(
+    event => {
+      switch (event.key) {
+        case 'ArrowUp':
+          if (selectedCommitIndex !== null) {
+            selectCommitIndex(
+              selectedCommitIndex > 0
+                ? selectedCommitIndex - 1
+                : commitIndices.length - 1,
+            );
+          }
```

Comment:
Oooh, this isn't right. I didn't catch it during my first review, but the selected component might not be in every commit. We'll need to filter the list.